continuous_integration/scripts/test_report.py

```diff
@@ -172,7 +173,7 @@ def dedup(group):
         for w in workflows
         if (
             pandas.to_datetime(w["created_at"])
-            > pandas.Timestamp.now(tz="UTC") - pandas.Timedelta(days=90)
+            > pandas.Timestamp.now(tz="UTC") - pandas.Timedelta(days=60)
```

Comment:
why reduce this?

Reply:
I'm starting to be concerned about bumping up against GitHub's rate limit for the use of `GITHUB_TOKEN` in actions (currently 1000 requests per hour). Roughly speaking, we are downloading ~10 artifacts per workflow run, plus a slew of administrative requests here. An alternative would be to just choose the most recent ~60-80 runs or thereabouts.

Reply:
ok, got it. 

I think what we're looking for is indeed "last X runs" and not "last X days". The way I understand this code, that would be very easy to change, wouldn't it?